install.sh

```diff
@@ -49,16 +52,68 @@ else
 	fi
 fi
 
+
+allBuildArguments=("$@")
+for ((index=0; index < ${#allBuildArguments[@]}; index++)); do
+	if [ ${allBuildArguments[index]} = '-r' ]; then
+		reposArg=${allBuildArguments[index+1]}
+		if [ "$reposArg" = "" ]; then
+			echo "Missing required argument for -r"
+			exit 1 # exit script with error
+		fi
+		# parse the r option values
+		IFS=',' read -r -a repos <<< "$reposArg"
+	fi
+done
+
 if [ -d ../operator ]; then 
 
 	# pluck image tag off operator image 
 	tag=$(cat ../operator/kappnav.yaml | grep operator: | awk '{ split($0,p,":"); print p[3] }')
-
+	
 	echo Install kappnav to kubeenv $kubeenv
 	kubectl create namespace kappnav 
 
-	cat ../operator/kappnav.yaml | sed "s|kubeEnv: okd|kubeEnv: $kubeenv|" | sed "s|repository: kappnav/|repository: $org/kappnav-|" | sed "s|tag: $tag|tag: latest|" | sed "s|image: kappnav/operator:$tag|image: $org/kappnav-operator:latest|" | kubectl create -f - -n kappnav 
+	if [[ x$reposArg == 'x' || x$reposArg == "" ]]; then
+		cat ../operator/kappnav.yaml | sed "s|kubeEnv: okd|kubeEnv: $kubeenv|" | sed "s|repository: kappnav/|repository: $org/kappnav-|" | sed "s|tag: $tag|tag: latest|" | sed "s|image: kappnav/operator:$tag|image: $org/kappnav-operator:latest|" | kubectl create -f - -n kappnav 
+	else
+		org=$DOCKER_USER
+		echo $DOCKER_PWD | docker login docker.io -u $DOCKER_USER --password-stdin
+		. ./projectList.sh
+		projs=$BUILD_PROJECTS
+		
+		cat ../operator/kappnav.yaml | \
+			sed "s|kubeEnv: okd|kubeEnv: $kubeenv|" | \
+			sed "s|tag: $tag|tag: dev|" | \
+			sed "s|image: kappnav/operator:$tag|image: kappnav/operator:dev|" \
+			> temp-kappnav.yaml
+		for p in $projs; do
```

Comment:
The outer loop isn't doing anything and isn't needed.  Please delete it (lines 90, 92, 108, 110).